defactor-com/ui-kit
# Feat/fix format and loan to ratio 85

## Add Card Component and allow to use string values

### What does this PR do?

- Resolve #85
  - Add Card Component to show a Card with fluctuation or color
- Resolve #86
   - Allow to use string to delegate the formatting values independently
   - To format chart number use the formatValue function
- Add the fluctuation changed value
- Fix missing ChartContainer export

### Steps to test

1. Run the project locally
2. Go to Dashboard component
3. Check the examples

### Checklist


### Screenshots

**Dashboard Component**

![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/5db5f762-3dc6-4111-af5a-efdd50ddf54b)
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/70ac3db2-1c41-4e9b-bf70-55c70bf74bbc)
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/59cbb49c-eab4-46a6-a138-4b839c845ec9)

**Card Component**

![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/cb1f5354-2bb6-4fb3-8004-9b1d376dc59d)
